### ISSUE TITLE
IE 11 support

### ISSUE THREAD

**Issue by Teebo** [zefoy/ngx-swiper-wrapper#263]
**IE version 11 and Edge**
It does not work on Internet Explorer when testing on **localhost:4200**. But works on production hosted site.

It does not work on **Edge** for both local host and production site
But it works in Edge: 80

Angular version: 9.0.6
I am getting the error
```
SCRIPT1002: Syntax error
vendor.js (46943,1)
```

**"swiper": "^5.3.6"**

The version of **ngx-swiper-wrapper** I am using is
`"ngx-swiper-wrapper": "^9.0.1",`

It works when using **ngx-swipper-wrapper: 8.0.2** but it breaks my view layout as some feature that are in 9.0.1 are not in the version 8.0.2.

Is it something I am missing?

**Comment by Cuberice** [comment]
Same issue after upgrading to Angular 9. There are multiple threads with no proper solution, at least not for Angular that I could find.

[Swiper](https://github.com/nolimits4web/swiper/issues/3199)
[Swiper Issue just Closed](https://github.com/nolimits4web/swiper/issues/2670)
[Another Issue](https://github.com/nolimits4web/swiper/issues/2282)
[On React](https://github.com/kidjp85/react-id-swiper/issues/273)

**Comment by Thabo-ops** [comment]
@Cuberice Yeah... the Dom7 issue, seems like the author of the package should do this in their end.
But in any case please let me know if you have found a solution

**Comment by Teebo** [comment]
@Cuberice  Yeah... the Dom7 issue, seems like the author of the package should do this in their end.
But in any case please let me know if you have found a solution

**Comment by edthenet** [comment]
I also wan't to know when there is a solution

**Comment by jhonnathanH** [comment]
same issue. with the version 
 `"ngx-swiper-wrapper": "^9.0.1"`
` "@angular/core": "~9.0.6"`

**Comment by jhonnathanH** [comment]
People i find a solution, reading the github page i see the demo on stackblitz and i analized the code and see thri thing that i miss in my project
1 install swiper `npm i swiper@5.3.7`
2 install types swiper  `npm i@types/swiper@5.3.0`
3 copy this lines in tsconfig.json
`... "lib": [
      "es2018",
      "dom"
    ],
    "paths": {
      "ngx-swiper-wrapper": [
        "dist/lib"
      ]
    }...` 
and dont forget "target": "es2015"

**Comment by Teebo** [comment]
@jhonnathanH does it work on Microsoft Edge for you?

**Comment by aakici** [comment]
> People i find a solution, reading the github page i see the demo on stackblitz and i analized the code and see thri thing that i miss in my project
> 1 install swiper `npm i swiper@5.3.7`
> 2 install types swiper `npm i @types/swiper@5.3.0`
> 3 copy this lines in tsconfig.json
> `... "lib": [ "es2018", "dom" ], "paths": { "ngx-swiper-wrapper": [ "dist/lib" ] }...`
> and dont forget "target": "es2015"

This didn't help me, I had downgrade to version 8.0.2.

**Comment by Teebo** [comment]
@aakici did the downgrade help you, how did you solve the issue?

**Comment by aakici** [comment]
@Teebo I just downgrading to version 8.0.2 and that's it. It solved my issue with IE11.

**Comment by Teebo** [comment]
@aakici OK I see, have you tested it on edge as well?

**Comment by aakici** [comment]
I didn't have any issues with Edge and it is still working after downgrade.

**Comment by Teebo** [comment]
@aakici OK I see, thank you

**Comment by edthenet** [comment]
So downgrading is a work arround, but it's not the solution

**Comment by jhonnathanH** [comment]
> @jhonnathanH does it work on Microsoft Edge for you?

you're right, now it fails in EDGE, if I change es2015 to es5 .. EDGE works fine but...  IE11 fail. I do not know what else to do.

**Comment by Teebo** [comment]
@jhonnathanH yes this is very peculiar, I am not sure how to solve this one

**Comment by vltansky** [comment]
According to https://github.com/nolimits4web/swiper/issues/3199#issuecomment-531185060 I've succeed to config angulars webpack to transpile dom7 to es5.

1. `npm i @angular-builders/custom-webpack`
2. in `angular.json`:
```json
...

      "architect": {
        "build": {
          "builder": "@angular-builders/custom-webpack:browser",
            "customWebpackConfig": {
              "path": "./extra-webpack.config.js",
              "mergeStrategies": {
                "externals": "replace"
              }
...
```
3. extra-webpack.config.js:
```js
const path = require("path");
module.exports = (config, options) => {
    config.module.rules.push({
      test: /\.(js|jsx)$/,
      use: {
        loader: 'babel-loader',
        options: {
          presets: ['@babel/preset-env']
        }
      },
      include: [
        path.resolve(__dirname, "node_modules/swiper"),
        path.resolve(__dirname, "node_modules/dom7"),
        path.resolve(__dirname, "node_modules/ssr-window"),
      ]
    })
    return config;
  };
```

**Comment by Teebo** [comment]
@vltansky bro you a star! That worked good

**Comment by PJMS** [comment]
@vltansky 's solution also worked for me! Thanks!
In my case, the `"customWebpackConfig"` had to be under `"options"` object in order to work as intended.

Angular 9.1.9

ref: https://github.com/just-jeb/angular-builders/tree/master/packages/custom-webpack#for-example

**Comment by vltansky** [comment]
> @vltansky 's solution also worked for me! Thanks!
> In my case, the `"customWebpackConfig"` had to be under `"options"` object in order to work as intended.
> 
> Angular 9.1.9
> 
> ref: https://github.com/just-jeb/angular-builders/tree/master/packages/custom-webpack#for-example

You're right, `"customWebConfig"` have to be under `"options"`. I've updated the instructions. Thanks.

**Comment by miladj3** [comment]
> > People i find a solution, reading the github page i see the demo on stackblitz and i analized the code and see thri thing that i miss in my project
> > 1 install swiper `npm i swiper@5.3.7`
> > 2 install types swiper `npm i @types/swiper@5.3.0`
> > 3 copy this lines in tsconfig.json
> > `... "lib": [ "es2018", "dom" ], "paths": { "ngx-swiper-wrapper": [ "dist/lib" ] }...`
> > and dont forget "target": "es2015"

this is works for me
